templates/travel/pages/index.js

```diff
@@ -36,492 +42,81 @@ export default function Index(props) {
           href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700'
           rel='stylesheet'
         />
-        <link rel='stylesheet' type='text/css' href='/css/travel.css' />
+        {/* TODO: This CSS file reference needs to be removed */}
+        <link rel='stylesheet' type='text/css' href='/css/travel.css' />{' '}
       </Head>
 
       {/* Put in a div so can collapse these easily in developer tools */}
-      <div>
-        <AmpState id='ui_hero'>{true}</AmpState>
-        <AmpState id='ui_reset'>{false}</AmpState>
-        <AmpState id='ui_filterPane'>{false}</AmpState>
-        <AmpState id='ui_sortPane'>{false}</AmpState>
-
-        <AmpState id='fields_query'>{''}</AmpState>
-        <AmpState id='fields_query_initial'>{''}</AmpState>
-        <AmpState id='fields_query_live'>{''}</AmpState>
-        <AmpState id='fields_query_edited'>{false}</AmpState>
-        <AmpState id='query_query'>{''}</AmpState>
-
-        <AmpState id='fields_departure'>{''}</AmpState>
-        <AmpState id='fields_departure_initial'>{''}</AmpState>
-        <AmpState id='fields_departure_edited'>{false}</AmpState>
-        <AmpState id='query_departure'>{''}</AmpState>
-
-        <AmpState id='fields_return'>{''}</AmpState>
-        <AmpState id='fields_return_initial'>{''}</AmpState>
-        <AmpState id='fields_return_edited'>{false}</AmpState>
-        <AmpState id='query_return'>{''}</AmpState>
-
-        <AmpState id='fields_type'>{[]}</AmpState>
-        <AmpState id='fields_type_initial'>{[]}</AmpState>
-        <AmpState id='fields_type_edited'>{false}</AmpState>
-        <AmpState id='query_type'>{[]}</AmpState>
-
-        <AmpState id='fields_city'>{[]}</AmpState>
-        <AmpState id='fields_city_initial'>{[]}</AmpState>
-        <AmpState id='fields_city_edited'>{false}</AmpState>
-        <AmpState id='query_city'>{[]}</AmpState>
-
-        <AmpState id='fields_sort'>{'popularity-desc'}</AmpState>
-        <AmpState id='fields_sort_initial'>{'popularity-desc'}</AmpState>
-        <AmpState id='fields_sort_edited'>{false}</AmpState>
-        <AmpState id='query_sort'>{'popularity-desc'}</AmpState>
-
-        <AmpState id='fields_maxPrice'>{801}</AmpState>
-        <AmpState id='fields_maxPrice_initial'>{801}</AmpState>
-        <AmpState id='fields_maxPrice_live'>{801}</AmpState>
-        <AmpState id='fields_maxPrice_edited'>{false}</AmpState>
-        <AmpState id='query_maxPrice'>{801}</AmpState>
-      </div>
+      <AmpState id='display'>
```

Comment:
I'm not a fan of namespacing via name as it results in unnecessary redundancy. Instead I'd prefer:

```
<AmpState id="ui">{{
   hero: true,
   reset: false,
  ...
}}<AmpState/>
```